packages/app/src/cli/models/organization.ts

```diff
@@ -18,6 +18,12 @@ export interface MinimalAppIdentifiers {
   organizationId: string
 }
 
+export interface MinimalAppIdentifiersPossiblyExcludingId {
```

Comment:
Are we using `MinimalAppIdentifiers` anywhere now? Should this be the one with the nice short name and `MinimalAppIdentifiers` be the one with the long obscure name?